src/elements/_has-arrow.scss

```diff
@@ -0,0 +1,279 @@
+/// Positioning with Arrows (tooltitps, popovers, etc.)
+@import "vars";
+
+/// ------------------------------------------------ ///
+/// -------------------- CONFIG -------------------- ///
+/// ------------------------------------------------ ///
+
+// Distance from tip to nearest tooltip corner
+$arrow-offset: 1.25rem; // 20px
+
+// length of one side of the square arrow container
+// 13px chosen because the hypotenuse calculates closest to 16px (per spec)
+$arrow-box-side: 13px;
+
+// Calculate long side (i.e., the hypotenuse of the arrow container)
+//$arrow-hypotenuse: sqrt(2 * $arrow-box-side * $arrow-box-side); // 18.3848px
+// sqrt(2 * boxSide * boxSide) = sqrt(2 * 13 * 13) = sqrt(338) = 18.3848px
+$arrow-hypotenuse: 18.3848px;
```

Comment:
Originally, this value was being calculated via the `sqrt()` and `pow()` LESS functions, but SASS doesn't have these functions built-in. However, this value never changed, so we replaced the calculation with the raw results.

There's no need to worry about this getting out of sync with design specifications any more than it normally would, because values would have to be updated, regardless.